src/plugins/intel_cpu/src/emitters/x64/jit_bf16_emitters.hpp

```diff
@@ -75,8 +75,10 @@ class jit_uni_vcvtneps2bf16 : public jit_emitter {
             h->uni_vpackusdw(aux, aux, aux);
 
             if (host_isa_ == dnnl::impl::cpu::x64::cpu_isa_t::avx2) {
+                h->vpermq(Ymm(aux.getIdx()), Ymm(aux.getIdx()), 0xD8); //11 01 10 00
                 h->vextracti128(out, Ymm(aux.getIdx()), 0);
             } else {
+                h->pshufd(Xmm(aux.getIdx()), Xmm(aux.getIdx()), 0xD8); //11 01 10 00
```

Comment:
Is this line really needed? as we are fixing cross lane alignment.

Reply:
SSE4.1 should also have this problem, but float->bf16 tests are disabled in non-avx512.

Reply:
I don't think it's needed on sse41, 4 bf16 data already locate the lower half of XMM at this point, as I mentioned the bug exist cross lane alignment, there is single lane on sse41.

Reply:
oh, I see, SSE4.1 use Xmm reg, which is 128bit, and the lower 64bit has got 4 bf16 data.

Reply:
I verify SSE4.1 branch with Intel(R) Penryn CPU, with or without this line the float to bf16 all pass, let's delete this line.